caliban_toolbox/utils/crop_utils.py

```diff
@@ -49,8 +52,23 @@ def compute_crop_indices(img_len, crop_size, overlap_frac):
         int: number of pixels of padding at start and end of image in given dimension
     """
 
-    # compute overlap fraction in pixels
-    overlap_pix = math.floor(crop_size * overlap_frac)
+    # compute indices based on fixed number of pixels per crop
+    if crop_size is not None:
+
+        # compute overlap fraction in pixels
+        overlap_pix = math.floor(crop_size * overlap_frac)
+
+    # compute indices based on fixed number of crops
+    elif num_crops is not None:
+        # number of pixels in non-overlapping portion of crop
+        non_overlap_crop_size = np.ceil(img_len / num_crops)
+
+        # Technically this is the fraction the non-overlap, rather than fraction of the whole,
+        # but we're going to visually crop overlays anyway to make sure value is appropriate
+        overlap_pix = math.floor(non_overlap_crop_size * overlap_frac)
+
+        # total crop size
+        crop_size = non_overlap_crop_size + overlap_pix
 
```

Comment:
Add case to explicitly catch if both `crop_size` and `num_crops` are `None`?

Reply:
Yup, totally forgot to add this in!